Scripts/netlify_redirects.txt

```diff
@@ -1,2 +1,2 @@
 /                      https://blog.cocoapods.org/CocoaPods-1.7.2/ 302
-/Specs/*               https://raw.githubusercontent.com/CocoaPods/Specs/master/Specs/:splat
+/Specs/*               https://cdn.jsdelivr.net/cocoa/Specs/:splat
```

Comment:
is this a special CDN not using the standard `/gh/CocoaPods/` scheme? is this setup in an account we have access to?

Reply:
It's a special URL because jsDelivr wanted to apply a special CDN config to it. I don't know much beyond the special TTLs.

It was created after I did some stress-testing  on the standard ones and they didn't like the loads.

Reply:
I got an assurance that the TTL on json files would be 1 hour and if that hurts them too much they'll raise it. Ideally that only affects deprecation latency.